test/specs/WebexMeeting/WebexMeeting.e2e.js

```diff
@@ -1,10 +1,46 @@
-describe('my widget', () => {
-  it('should find sections and match text', () => {
-    browser.url('https://webex.github.io/widgets/');
+import {createUser, removeUser} from '../../util';
+import MeetingPage from './MeetingPage';
+
+describe('Meeting Widget', () => {
+  let user;
+  let room;
+  let accessToken;
+  let meetingDestination;
+
+  beforeAll(async () => {
+    try {
+      user = await createUser();
+      room = await user.sdk.rooms.create({title: 'Test Room'});
+    } catch (e) {
+      console.error(e);
+    }
+
+    accessToken = process.env.WEBEX_ACCESS_TOKEN || (user && user.token.access_token);
+    meetingDestination = process.env.WEBEX_MEETING_DESTINATION || (room && room.id);
+
+    expect(accessToken).toBeDefined();
+    expect(meetingDestination).toBeDefined();
+  });
+
+  afterAll(async () => {
+    if (user) {
+      if (room) {
+        await user.sdk.rooms.remove(room).catch(console.error);
+      }
+      removeUser(user);
+    }
+  });
+
+  it('opens meeting test page', () => {
+    MeetingPage.open();
 
     // should contain two sections
     expect($$('.content > section').length).toBe(2);
     // should match first section text
     expect($('.content > section > h3')).toHaveTextContaining('Webex Widgets Access Token');
   });
+
+  it('loads meeting widget', () => {
```

Comment:
```suggestion
  it('loads, () => {
```

Currently it will read "Meeting Widget loads meeting widget", it should read "Meeting widget loads"